interface/ceed-basis.c

```diff
@@ -780,6 +780,32 @@ int CeedBasisIsTensor(CeedBasis basis, bool *is_tensor) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Determine if given `CeedBasis` has nodes collocated with quadrature points
+
+  @param[in]  basis     `CeedBasis`
+  @param[out] is_tensor Variable to store collocated status
```

Comment:
```suggestion
  @param[out] is_collocated Variable to store collocated status
```

Reply:
fixed via manual commit and rebase as this forces spacing changes